packages/react/src/side-nav/SideNav.test.tsx

```diff
@@ -41,17 +41,35 @@ describe('SideNav', () => {
 	describe('renders the title correctly', () => {
 		test('with link', () => {
 			renderSideNav(defaultTestingProps);
-			const el = screen.getByText(defaultTestingProps.title);
-			expect(el.tagName).toBe('A');
+
+			const el = screen.getByRole('link', {
+				name: defaultTestingProps.title,
+				hidden: true,
+			});
+
+			expect(el).toBeInTheDocument();
```

Comment:
Shouldn't need this assertion. If it's not in the document, line 45 will fail the test and the next line is a good assertion.

Reply:
Don't want to make any more changes in case the tests fail 🤣

Reply:
Lol no arguments there. Let's get this merged while the test gods are happy.